### ISSUE TITLE
Restore "Add ping pong health"

### ISSUE THREAD

**Issue by franknoirot** [KittyCAD/modeling-app#1770]
Commit SHA: 61d7950ca37ed6eb59dade1161324f4c4c847d98

Restore this feature by fixing the remaining issues found with it in the desktop Tauri app, which result in a consistent "Stream disconnected" state after navigating from any project to home.

**Comment by jessfraz** [comment]
why not just hide the health thing on the home screen?

**Comment by jessfraz** [comment]
seems like the easiest fix

**Comment by franknoirot** [comment]
Yeah it is from the appearance side, and when we restore this we should just remove the NetworkHealthIndicator from the AppHeader to avoid that issue. But the more functional issue is that after that first navigation back to home, the stream was showing a "Stream disconnected" message with a loading spinner whenever you would navigate back into a project, which would stay there and was only recoverable by refreshing the page.

**Comment by franknoirot** [comment]
> Yeah it is from the appearance side, and when we restore this we should just remove the NetworkHealthIndicator from the AppHeader to avoid that issue. But the more functional issue is that after that first navigation back to home, the stream was showing a "Stream disconnected" message with a loading spinner whenever you would navigate back into a project, which would stay there and was only recoverable by refreshing the page.

This is no longer happening. @lf94  do you need to do anything else for this or can I close?

**Comment by lf94** [comment]
We can close this because we're not bringing it back. We're introducing a new "ping" metric and nothing more.
